src/components/c-google-maps.vue

```diff
@@ -15,6 +15,44 @@
 
   let geocoder;
 
+  let isMapsAPILoaded = false;
+
+  const callbackStack = [];
+
+  window.cGoogleMapsInitMap = () => {
+    isMapsAPILoaded = true;
+
+    callbackStack.forEach(callback => callback());
+
+    callbackStack.length = 0;
+  };
+
+  /**
+   * This function checks whether the Map API has already been loaded once.
+   *
+   * @param {Function} callback - Callback method that is called when the Map API has already been loaded successfully.
+   */
+  const loadMapsAPI = (callback) => {
```

Comment:
todo: please use a function instead, so we don't need to define a variable/const.

```suggestion
  function loadMapsAPI(callback) {
```